docs/data-ai/ai/create-dataset.md

```diff
@@ -14,244 +14,169 @@ aliases:
-                           x_min_normalized=box["xMinNormalized"],
-                           y_min_normalized=box["yMinNormalized"],
-                           x_max_normalized=box["xMaxNormalized"],
-                           y_max_normalized=box["yMaxNormalized"]
-                       )
-
-               file_ids.append(binary_id)
-
-       await data_client.add_binary_data_to_dataset_by_ids(
-           binary_ids=file_ids,
-           dataset_id=dataset_id
-       )
-       print("Added files to dataset.")
-       print("https://app.viam.com/data/datasets?id=" + dataset_id)
-
-       viam_client.close()
-
-   if __name__ == '__main__':
-       asyncio.run(main())
-   ```
+To train a model, you need a dataset that meets the following criteria:
 
-1. Run the script to upload the images and their metadata into a dataset in Viam app providing the following input:
+- the dataset contains at least 15 images
+- at least 80% of the images have labels
+- for each selected label, at least 10 bounding boxes exist
 
-   ```sh {class="command-line" data-prompt="$" }
-   python upload_data.py -org-id <ORG-ID> -api-key <API-KEY> \
-      -api-key-id <API-KEY-ID> -machine-part-id <MACHINE-PART-ID> \
-      -location-id <LOCATION-ID> -dataset-name <NAME>
-   ```
+This page explains how to create a dataset that meets these criteria for your training purposes.
+
+## Prerequisites
+
+{{% expand "A machine connected to the Viam app" %}}
 
-1. Continue to [Train a tflite machine learning model](/data-ai/ai/train-tflite/).
+{{% snippet "setup.md" %}}
+
+{{% /expand %}}
+
+{{% expand "A camera, connected to your machine, to capture images" %}}
+
+Follow the guide to configure a [webcam](/operate/reference/components/camera/webcam/) or another [camera component](/operate/reference/components/camera/), if you haven't already.
 
 {{% /expand%}}
 
-## Prerequisites
+## Capture images
+
+{{< tabs >}}
+{{% tab name="Capture individual images directly into a dataset" %}}
+
+You can add images to a dataset directly from a camera or vision component feed in the **CONTROL** or **CONFIGURATION** tabs of the Viam app.
+This technique can help you create a dataset without filling the **DATA** tab with a large number of unhelpful empty images.
+
+To add an image directly from a dataset from a visual feed, complete the following steps:
+
+1. Open the **TEST** panel of any camera or vision component to view a feed of images from the camera.
+1. Click the button marked with the camera icon to save the currently displayed image to a dataset:
+   {{< imgproc src="/components/camera/add_image_to_dataset_button.png" alt="A button marked with the outline of a camera, emphasized in red" resize="800x" style="width:500px" class="imgzoom" >}}
+1. Select an existing dataset.
+1. Click **Add** to add the image to the selected dataset.
+1. When you see a success toast that reads "Saved image to dataset", you have successfully added the image to the dataset.
```

Comment:
I don't know if people generally understand what a toast is...
```suggestion
1. When you see a success notification that reads "Saved image to dataset", you have successfully added the image to the dataset.
```

Reply:
maybe popop? Or maybe just message?

Reply:
I like notification -- it looks quite similar to notifications on the desktop. Message is probably the most accurate and futureproof, but lacks some of the descriptive punch of *notification*.